pillar/heroku/ocw-studio.sls

```diff
@@ -77,6 +77,7 @@ heroku:
     AWS_STORAGE_BUCKET_NAME: 'ol-ocw-studio-app-{{ env_data.env }}'
     CONTENT_SYNC_BACKEND: content_sync.backends.github.GithubBackend
     CONTENT_SYNC_PIPELINE: content_sync.pipelines.concourse.ConcourseGithubPipeline
+    CONCOURSE_URL: {{ env_data.CONCOURSE_URL }}
     CONCOURSE_USERNAME: oldevops
     CONCOURSE_PASSWORD: __vault__::secret-concourse/web>data>data>admin_password
```

Comment:
This doesn't seem to be carried over to heroku, any idea why?  Is the value missing from vault?

Reply:
It's because the path isn't quite right. It needs to be `secret-concourse/data/web`

Reply:
Thanks, just changed it to ```
CONCOURSE_PASSWORD: __vault__::secret-concourse/data/web>data>data>admin_password
```